data/sql/updates/pending_db_world/rev_1641422836099367029.sql

```diff
@@ -0,0 +1,326 @@
+(60903, 17190, 530, 0, 0, 1, 1, 0, 0, -5345.604, -10915.708, -0.6951377, 3.472287893295288085, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -5345.604 -10915.708 -0.6951377
+(60904, 17190, 530, 0, 0, 1, 1, 0, 0, -5348.8247, -10881.534, -1.4308957, 0.935154259204864501, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -5348.8247 -10881.534 -1.4308957
+(60905, 17192, 530, 0, 0, 1, 1, 0, 0, -5326.346, -10924.387, -0.061333664, 3.229621648788452148, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -5326.346 -10924.387 -0.061333664
+(60906, 17190, 530, 0, 0, 1, 1, 0, 0, -5350.43, -11013.511, 0.35168836, 0.324238121509552001, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -5350.43 -11013.511 0.35168836
+(60907, 17190, 530, 0, 0, 1, 1, 0, 0, -5350.531, -10987.597, -1.6087946, 4.186498641967773437, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -5350.531 -10987.597 -1.6087946
+(60908, 17190, 530, 0, 0, 1, 1, 0, 0, -5419.6704, -10952.906, -21.553621, 0.472459375858306884, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -5419.6704 -10952.906 -21.553621
+(60909, 17192, 530, 0, 0, 1, 1, 0, 0, -5426.0225, -11008.705, -23.93589, 3.582702398300170898, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -5426.0225 -11008.705 -23.93589
+(60910, 17191, 530, 0, 0, 1, 1, 0, 0, -5415.7188, -10981.389, -21.267178, 0.531979978084564208, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -5415.7188 -10981.389 -21.267178
+(60911, 17190, 530, 0, 0, 1, 1, 0, 0, -5450.788, -10978.354, -38.85598, 0.367103815078735351, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -5450.788 -10978.354 -38.85598
+(60912, 17191, 530, 0, 0, 1, 1, 0, 0, -5450.012, -10954.308, -38.900898, 1.450325965881347656, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -5450.012 -10954.308 -38.900898
+(60913, 17190, 530, 0, 0, 1, 1, 0, 0, -5451.4165, -11014.778, -32.491295, 1.298519492149353027, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -5451.4165 -11014.778 -32.491295
+(60914, 17191, 530, 0, 0, 1, 1, 0, 0, -5476.1006, -10987.348, -153.56447, 2.300004959106445312, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -5476.1006 -10987.348 -153.56447
+(60915, 17191, 530, 0, 0, 1, 1, 0, 0, -5478.5923, -11013.903, -99.875465, 3.014854192733764648, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -5478.5923 -11013.903 -99.875465
+(60916, 17190, 530, 0, 0, 1, 1, 0, 0, -5478.1235, -10953.604, -151.27324, 3.179136276245117187, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -5478.1235 -10953.604 -151.27324
+(60917, 17192, 530, 0, 0, 1, 1, 0, 0, -4773.0347, -10936.996, 0.79550934, 2.093307971954345703, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -4773.0347 -10936.996 0.79550934
+(60918, 17192, 530, 0, 0, 1, 1, 0, 0, -5422.1284, -10983.814, -25.117788, 0.890117883682250976, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0); -- .go xyz -5422.1284 -10983.814 -25.117788
```

Comment:
remove all the end of lines notes or move them to above each query line for it to pass the testing.